pkg/networkmanager/interfaces.js

```diff
@@ -2596,6 +2612,19 @@ PageNetworkInterface.prototype = {
                                                        dev.DeviceType == 'bridge'));
         $('#network-interface-delete').toggle(is_deletable && managed);
 
+        function render_interface_section_separator(title) {
+            return $('<td class="network-interface-separator" colspan="100%">').text(title);
+        }
+
+        function render_interface_type_row() {
+            if (dev) {
+                return $('<tr>').append(
+                    $('<td>').text(_("Type")),
```

Comment:
I don't think we need this extra row. We don't generally want to show the connection id, and interface and device type are already in the panel heading.